### PR TITLE
feat/redis

### PR DESCRIPTION
**Pull request by disturbing**
This change is [Reviewable](https://reviewable.io/reviews/kintohub/charts/7)

### CODE REVIEW

**Comment by disturbing** [comment]
image.repository=redis
image.tag=5.0.5-alpine
replicas=3
auth=false
redisPassword
persistentVolume.enabled=false
persistentVolume.size=10Gi

**Comment by bakayolo** [comment]
For every helm chart, suggest we add the kintohub variables
```
image.repository=redis
image.tag=5.0.5-alpine
replicas=3
auth=false
redisPassword
persistentVolume.enabled=false
persistentVolume.size=10Gi
```

in a file `values.kinto` so that anyone can see it and modify them if needed

File: stable/redis-ha/templates/NOTES.txt [hunk 1]
```diff
@@ -1,25 +1,2 @@
...
-{{- else }}
-1. Run a Redis pod that you can use as a client:
-
-   kubectl exec -it {{ template "redis-ha.fullname" . }}-server-0 sh -n {{ .Release.Namespace }}
-
-2. Connect using the Redis CLI:
-
-  redis-cli -h {{ template "redis-ha.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
-{{- end }}
+{{ template "redis-ha.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
```
Comment (bakayolo) [inline stable/redis-ha/templates/NOTES.txt]: Remove `.{{ .Release.Namespace }}.svc.cluster.local`
No need and it's too much information.

File: stable/redis-ha/templates/redis-ha-statefulset.yaml [hunk 2]
```diff
@@ -78,8 +82,6 @@ spec:
                     release: {{ .Release.Name }}
                 topologyKey: failure-domain.beta.kubernetes.io/zone
     {{- end }}
-      securityContext:
-{{ toYaml .Values.securityContext | indent 8 }}
```
Comment (bakayolo) [inline stable/redis-ha/templates/redis-ha-statefulset.yaml]: Leaving it empty should have done the job but ok

**Review by bakayolo — CHANGES_REQUESTED** [review]

**Comment by bakayolo** [comment]
Missing resources limit

**Pull request merged**